libs/stats/odc/stats/_fc_percentiles.py

```diff
@@ -0,0 +1,115 @@
+"""
+Fractional Cover Percentiles
+"""
+from functools import partial
+from typing import Optional, Tuple
+from itertools import product
+import dask.array as da
+import xarray as xr
+import numpy as np
+from odc.stats.model import Task
+from odc.algo.io import load_with_native_transform
+from odc.algo import keep_good_only
+from odc.algo._percentile import xr_percentile
+from odc.algo._masking import _fuse_with_custom_op, _or_fuser, _first_valid_np, _fuse_or_np
+from .model import StatsPluginInterface
+from . import _plugins
+
+
+class StatsFCP(StatsPluginInterface):
+    
+    NAME = "ga_fc_percentiles"
+    SHORT_NAME = NAME
+    VERSION = "0.0.1"
+    PRODUCT_FAMILY = "fc_percentiles"
+
+    def __init__(
+        self,
+        resampling: str = "bilinear",
+    ):
+        self.resampling = resampling
+
+    @property
+    def measurements(self) -> Tuple[str, ...]:
+        _measurments = [f"{b}_pc_{p}" for b, p in product(["pv", "bs", "npv", "ue"], ["10", "50", "90"])]
+        _measurments.append("is_ever_wet")
```

Comment:
didn't you rename it into `qa`

Reply:
I don't know what band is needed, probably check with Mathew, it looks like `is_ever_wet` boolean classification was one of the proposed mechanisms

Reply:
I had a chat w/ Mathew and an enum band would be preferred. I've chucked it up on slack to discuss w/ Mathew and Emma exactly what values should be included

Reply:
they wanted the enum order reversed but other than that all good!